packages/material/package.json

```diff
@@ -81,11 +81,11 @@
   "devDependencies": {
     "@jsonforms/core": "^2.3.2",
     "@jsonforms/react": "^2.3.2",
-    "@jsonforms/test": "^2.3.2",
```

Comment:
Are these changes meeded for the removal?

Reply:
Yes as we are also removing the test package